crates/rolldown/tests/rolldown/warnings/sourcemap_broken/load/artifacts.snap

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
rollup don't generate missing warning for load hook, please remove this testcase.